sale_commission/report/sale_commission_analysis_report.py

```diff
@@ -107,3 +107,104 @@ def init(self):
                 AsIs(self._group_by())
             ),
         )
+
+
+class SaleOrderCommissionAnalysisReport(models.Model):
```

Comment:
Better to put this on a new file, as they are separate models.

Reply:
done